packages/@o3r/pipeline/schematics/ng-add/index.ts

```diff
@@ -26,12 +26,15 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
     }
     context.logger.info(`Setting up pipeline for package manager: "${packageManager}" `);
     const setupCommand = packageManager === 'yarn' ? 'yarn install --immutable' : 'npm ci';
+    const isYarn2 = packageManager === 'yarn' && tree.exists('/.yarnrc.yml');
```

Comment:
`.yarnrc.yml` does exist for Yarn 1 as well

Reply:
actually, Yarn 1 is using `.yarnrc` and Yarn 2+ `yarnrc.yml`

Reply:
Not sure about that 
In local I have yarn berry 
when I have to work on yarn1 project I just run `yarn set version 1.xx.xx` and it creates me a `.yarnrc.yml` file
so yarn1 seems to work with `.yarnrc.yml` too

Reply:
Indeed, maybe the safest is to detect the version targeted by the `yarnPath` the `.yarnrc.yml` if it exists.
So we get something like:

- `.yarnrc.yml` does not exists -> `yarn1`
- `.yarnrc.yml` exists but no `yarnPath` -> `yarn2+`
- `.yarnrc.yml` exists `yarnPath` match `yarn-1` -> `yarn1`
- everything else -> `yarn2+`

Reply:
are you sure it is not creating a `.yarnrc` when using yarn 1? For me it creates a `.yarnrc`  and not `.yarnrc.yml`

Reply:
Updated the yarn version detection and added some tests